macros/src/main/scala/me/rexim/morganey/meta/package.scala

```diff
@@ -0,0 +1,19 @@
+package me.rexim.morganey
+
+import scala.language.experimental.macros
+
+package object meta extends AnyRef
+  with DefaultLiftableInstances
+  with DefaultUnliftableInstances {
+
+  private[meta] trait Quasiquotation {
+    def apply[T](args: T*): Any = macro QuotationMacro.quote
+    def unapply(arg: Any): Any = macro QuotationMacro.unquote
+  }
+
+  implicit class Quasiquote(ctx: StringContext) {
+    object m extends Quasiquotation
+    object lc extends Quasiquotation
```

Comment:
Yes, the lc interpolator does completely do the same thing as m. So if you want we could remove it. I thought it would nice to choose between which interpolator to use.